README.md

```diff
@@ -1,159 +1,170 @@
+- To run JUnit integration test of the complete flow from ingest to curation, use `./gradlew test`
+- To run MarkLogic Unit Tests (developed in server-side Javascript), use `./gradlew mlUnitTest`
 
-The test suites are found in project directories, here:
+The test suites can be found in the following project directories:
 
 - JUnit integration: `src/test/java/com/marklogic/hsk`
 - MarkLogic unit tests: `src/test/ml-modules/root/test/suites`
 
+## Maintaining and Modifying the HSK
+
+### Extending the HSK
 
-## Notes on Maintaining and Modifying the Healthcare Starter Kit (HSK)
+See the COOKBOOK for more information on how to extend the HSK
 
 ### About the sample source data
 
-The sample health population data provided in this project was generated using the Synthea synthetic health records project found here: [https://github.com/synthetichealth/synthea](https://github.com/synthetichealth/synthea)
+The sample health population data provided in this project was generated using the [Synthea synthetic health records project](https://github.com/synthetichealth/synthea)
 
 The HSK project provides sample records for 755 patients and associated healthcare providers, organizations, claims, claims transactions, and payors.
 
-### ml-gradle project
-
-The Marklogic Gradle plugin project is found here: [MarkLogic Gradle Plugin](https://github.com/marklogic-community/ml-gradle). This project provides the commands needed to deploy, maintain, test and modify the HSK project. Full documentation is found here: [MarkLogic Gradle Plugin Wiki](https://github.com/marklogic-community/ml-gradle/wiki)
+### ml-gradle
 
+The [Marklogic Gradle plugin](https://github.com/marklogic-community/ml-gradle) (ml-gradle) provides the commands needed to deploy, maintain, test and modify the HSK. Full documentation can be found on the [ml-gradle Wiki](https://github.com/marklogic-community/ml-gradle/wiki)
 
 ### Data Hub Central and ml-gradle
 
- **Data Hub Central** can be used to modify entities, run ingest and curation steps, explore content, and monitor jobs. Please note that using Data Hub Central, any changes are not propagated immediately to the local file directory that stores the artifacts for the base HSK project. You can use features of DHC to download them to your project directory, or this ml-gradle command: 
- - `./gradlew mlWatch -i -PenvironmentName=local`.  
+Data Hub Central (DHC) can be used to modify entities, run ingest and curation steps, explore content, and monitor jobs. Please note that when making changes using DHC, they are not propagated to the local project directory. You can run `./gradlew hubPullChanges` to download the changes made in DHC and write them to your local project directory.
+
+> `./gradlew hubPullChanges` will overwrite any local changes you have made that were not pushed to the database using `./gradlew hubDeployUserArtifacts`

```

Comment:
Suggest clarifying that "artifacts" will be overwritten from the database, but code and other configuration will not.